packages/experiments-realm/productivity/task.gts

```diff
@@ -696,242 +775,296 @@ export class Tag extends CardDef {
+      }
+      header {
         display: flex;
-        flex-wrap: wrap;
         justify-content: space-between;
         align-items: center;
       }
-      .task-detail {
-        min-height: var(--boxel-form-control-height);
-        margin-bottom: var(--boxel-sp-sm);
+      .task-title {
+        font-size: var(--boxel-font-size-med);
+        font-weight: 600;
       }
-      .task-meta {
-        display: flex;
-        flex-direction: column;
-        gap: var(--boxel-sp-xs);
+      .status-label {
+        font-size: var(--boxel-font-size-sm);
+        font-weight: 600;
+        margin-top: var(--boxel-sp-xs);
+      }
+      .text-gray {
+        color: var(--boxel-400);
+      }
+      .progress-bar-horizontal {
+        display: none;
+        --progress-bar-font-color: var(--boxel-dark);
+        border: 0px;
+        margin-top: var(--boxel-sp);
+      }
+      .task-divider.border-gray {
+        border: 1px solid var(--boxel-100);
+      }
+      .task-divider.border-white {
+        border: 1px solid var(--boxel-light);
+      }
+      .task-info {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        gap: var(--boxel-sp-xl);
+      }
+      .task-info .right-column {
+        display: grid;
+        gap: var(--boxel-sp-xl);
       }
       .task-assignee {
+        display: inline-flex;
         width: auto;
         height: auto;
         overflow: unset;
       }
-      .row-1 {
-        display: flex;
-        flex-wrap: wrap;
-        align-items: center;
-        gap: var(--boxel-sp-xxs);
-      }
-      .row-2 {
+      .task-tags {
         display: flex;
         flex-wrap: wrap;
-        gap: var(--boxel-sp-xxs);
-      }
-      .progress-bar-container {
-        --boxel-progress-bar-fill-color: var(--boxel-highlight);
-        width: 35%;
-        max-width: 400px;
-      }
-
-      .task-dates {
-        display: flex;
         align-items: center;
         gap: var(--boxel-sp-xxs);
-        color: var(--boxel-600);
+        overflow: hidden;
       }
-      .calendar-icon {
-        flex-shrink: 0;
+      .task-tag {
+        width: auto;
+        height: auto;
+        overflow: unset;
       }
       .date-info {
-        font-size: var(--boxel-font-size-xs);
+        font-size: calc(var(--boxel-font-size-xs) * 0.95);
         font-weight: 500;
         color: var(--boxel-600);
-        line-height: calc(var(--boxel-font-size-xs) * 0.9);
-      }
-      .children-header {
-        font-size: var(--boxel-font-size-sm);
-        color: var(--boxel-purple);
-        margin-bottom: var(--boxel-sp-xxs);
+        white-space: nowrap;
+        -webkit-line-clamp: 1;
+        overflow: hidden;
+        text-overflow: ellipsis;
       }
-      .subtasks-section {
-        border-radius: var(--boxel-border-radius);
-        min-width: 150px;
-        max-width: 600px;
+      .no-data-found-txt {
+        font-size: calc(var(--boxel-font-size-xs) * 0.95);
+        font-weight: 500;
```

Comment:
use one of our vars here

Reply:
Owh I see that u mentioned no css variable for font-weight. Noted @burieberry is there any possibility we can promote font-weight to a global variable. There are many places which use font-weight: 500 - 600. For now, I told  @lucaslyl to just put a var at the root of the isolated format. But styles are not shared across formats soo we r limited to that for now